package.json references a devDependancy that no longer exists

It looks like the devDependancy `expand-home-dir` had a new version published yesterday (0.0.3) and in the process, the publisher decided to remove the older 0.0.2 version (!?!?). 

Since package.json references the exact version for this lib, then npm install fails if either `--dev` is used, or another project references 'stormpath' as a dependancy in it's package.json:

```
$ npm install stormpath --dev
npm ERR! Darwin 15.2.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "stormpath" "--dev"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! version not found: expand-home-dir@0.0.2
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sking/tmp/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "benchmark": "^2.0.0",
     "chai": "~3.5.0",
     "coveralls": "^2.11.4",
-    "expand-home-dir": "0.0.2",
+    "expand-home-dir": "^0.0.3",
     "fake-fs": "^0.5.0",
     "grunt": "~0.4.2",
     "grunt-cli": "~0.1.13",

